### PR TITLE
Terraform: http uptime check alert

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: prow/oss/terraform/modules/alerts/main.tf [hunk 1]
```diff
@@ -133,3 +133,37 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
...
+      | condition val() > 1 '1'
+      EOT
+      trigger {
+        count = 1
+      }
+    }
+  }
+
+  documentation {
+    content   = "Host Down"
```
Comment (cjwagner) [inline prow/oss/terraform/modules/alerts/main.tf:163]: This is not going to be enough information to address the alert once we have more than one host being monitored. We need to include the URL of the host that is down somehow.

File: prow/oss/terraform/modules/alerts/main.tf [hunk 2]
```diff
@@ -133,3 +133,37 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
...
+  display_name = "HostDown"
+  combiner     = "OR"
+  conditions {
+    display_name = "Host is unreachable"
+    condition_monitoring_query_language {
+      duration = "120s"
+      query    = <<-EOT
+      fetch uptime_url
+      | metric 'monitoring.googleapis.com/uptime_check/check_passed'
+      | filter (metric.check_id == 'blackbox-exp')
```
Comment (cjwagner) [inline prow/oss/terraform/modules/alerts/main.tf:149]: What is this field and where does its value come from? I only see this value in the existing metrics.

File: prow/oss/terraform/modules/alerts/main.tf [hunk 3]
```diff
@@ -133,3 +133,37 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
...
+      | align next_older(1m)
```
Comment (cjwagner) [inline prow/oss/terraform/modules/alerts/main.tf:150]: The choice between `next_older` and `next_younger` doesn't matter here, right? You're just trying to grab a discrete value without actual aggregation?

File: prow/oss/terraform/main.tf [hunk 4]
```diff
@@ -42,4 +42,7 @@ module "alert" {
         "sinker" = {"namespace": "default"}
         "tide" = {"namespace": "default"}
     }
+    blackbox_probers = [
+        "oss-prow.knative.dev"
```
Comment (cjwagner) [inline prow/oss/terraform/main.tf:46]: I see an existing uptime check for `testgrid.k8s.io`, was that manually added for testing?

**Review by cjwagner — APPROVED** [review]
/hold
I just have nits

File: prow/oss/terraform/main.tf (hunk 4 above)
Comment (chaodaiG) [reply prow/oss/terraform/main.tf:46]: yep

File: prow/oss/terraform/modules/alerts/main.tf (hunk 3 above)
Comment (chaodaiG) [reply prow/oss/terraform/modules/alerts/main.tf:150]: > The choice between `next_older` and `next_younger` doesn't matter here, right? You're just trying to grab a discrete value without actual aggregation?

this was adapted from auto-generated alerts from UI, where we can set an alert from `uptime check` page

File: prow/oss/terraform/modules/alerts/main.tf (hunk 2 above)
Comment (chaodaiG) [reply prow/oss/terraform/modules/alerts/main.tf:149]: this was from the experiment, removed

File: prow/oss/terraform/modules/alerts/main.tf (hunk 1 above)
Comment (chaodaiG) [reply prow/oss/terraform/modules/alerts/main.tf:162]: Unfortunately this is not available, but I believe that the alert email will tell us more

**Comment by cjwagner** [comment]
LGTM. Ready to `/unhold` this? (`lgtm` is sticky)

**Pull request merged**